### PR TITLE
Add a get_exit_code() helper

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -213,6 +213,37 @@ function capture(
...
+        timeout: $timeout,
+        allowFailure: true,
+        context: $context,
+        quiet: $quiet,
+    );
+
+    $code = $process->getExitCode();
+
+    if ($code === null) {
+        throw new \LogicException('The process has not been terminated.');
```
Comment (lyrixx) [inline src/functions.php:241]: How is it possible?

File: doc/03-run.md [hunk 2]
```diff
@@ -144,6 +144,23 @@ function whoami()
...
+
+```php
+use Castor\Attribute\AsTask;
+
+use function Castor\get_exit_code;
+
+#[AsTask()]
+function whoami(): int
+{
+    $whoami = capture('whoami');
```
[the remark this answers is not in the document]
Comment (pyrech) [reply doc/03-run.md]: You looked at old code :)

File: src/functions.php (hunk 1 above)
Comment (pyrech) [reply src/functions.php:241]: Don't know but this is what the function signatures specifies

```
     * @return int|null The exit status code, null if the Process is not terminated```

Comment (lyrixx) [reply src/functions.php:241]: okay, but we know the process is finished... but okay.

Comment (pyrech) [reply src/functions.php:241]: I know, but I want to return an `int` not a nullable int. for better DX. What should I do in this case?

Comment (lyrixx) [reply src/functions.php:241]: `return $process->getExitCode()` But I understand you added this code for phpstan. Not a big deal

**Comment by lyrixx** [comment]
I think you forgot to rebuild the test suite since you added the example

**Comment by pyrech** [comment]
Done

**Review by lyrixx — APPROVED** [review]
Perfect, thanks 👍🏼

**Pull request merged**